fda/datasets/_real_datasets.py

```diff
@@ -440,10 +442,91 @@ def fetch_weather(return_X_y: bool = False):
     fetch_weather.__doc__ += _weather_descr + _param_descr
 
 _aemet_descr = """
-    Series of daily summaries of 73 spanish weather stations selected for the period 1980-2009.  The
-    dataset contains geographic information of each station and the average for the period 1980-2009
-    of daily temperature, daily precipitation and daily wind speed.
-    Meteorological State Agency of Spain (AEMET), http://www.aemet.es/. Government of Spain.
+    Series of daily summaries of 73 spanish weather stations selected for the period 1980-2009. The
+    dataset contains the average for the period 1980-2009 of daily temperature, daily precipitation and 
+    daily wind speed.
+    Below, the geographic information of each station is left.
+       
+    ======== ====== ================================= ======================== ===== ============ ============
```

Comment:
Why including it here instead of in the data? Is it not part of the original data?

Reply:
Where do I include it? It is not target, not metadata, so I think in the description is the best place.

Reply:
I would put it in metadata for the moment. I am not very proud of the dataset API, but I am waiting to see how the Scikit-learn developers add an alternative API based in Pandas, to imitate them.

Reply:
done